linetools/spectra/xspectrum1d.py

```diff
@@ -136,7 +145,9 @@ def normalize(self, conti, verbose=False, no_check=False):
                 raise ValueError('normalize: Continuum needs to be same length as flux array')
 
         # Adjust the flux
-        self.flux = self.flux / conti
+        self.flux /= conti
+        if self.uncertainty is not None:
+            self.uncertainty.array /= conti
```

Comment:
Should we have a flag that specifies whether the flux has been normalized previously, i.e. to avoid doing this step twice?
Or should we put that on the user.  I guess the latter is fine..